package-lock.json

```diff
@@ -0,0 +1,3298 @@
+{
+  "name": "vue-example-project",
+  "version": "0.0.0",
```

Comment:
I think the lock file needs updating since it’s displaying different version from the package.json

Reply:
Adjusted